.github/workflows/update-copyright-year.yml

```diff
@@ -0,0 +1,37 @@
+# https://github.com/FantasticFiasco/action-update-license-year
+# https://github.com/FantasticFiasco/action-update-license-year?tab=readme-ov-file#i-want-to-update-my-license-and-a-custom-source-in-the-same-pr
+
+name: Update copyright year(s) in license file
+
+on:
+  schedule:
+    - cron: '0 3 1 1 *' # 03:00 AM on January 1
+
+jobs:
+  update-license-year:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: FantasticFiasco/action-update-license-year@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@vprusso This is ready to be merged. You will have to set up a token in `settings` for this. 

I don't know if this works exactly as intended as this can't be triggered until Jan 1. The link for the action is added as a comment at the top of this file.